bitcoin/src/pow.rs

```diff
@@ -128,6 +116,27 @@ impl Target {
     // the low 208 bits are all zero.
     pub const MAX: Self = Target(U256(0xFFFF_u128 << (208 - 128), 0));
 
+    /// The maximum **attainable** target value on mainnet.
+    ///
+    /// Not all target values are attainable because consensus code uses the compact format to
+    /// represent targets (see `CompactTarget`).
```

Comment:
Why is the text for doc-comments different for mainnet and others. Should we not have uniform doc-comment across of all them? Can we also add some constants

Reply:
Oh, that's sloppy work by me. They should all be uniform. I added a note to #2107. At one stage I had all the constants from core with tests that when converted to compact form they matched these but I dropped it all to get this bug fix diff as small as possible.